alerter.c

```diff
@@ -2,6 +2,7 @@
 #include <assert.h>
 
 int alertFailureCount = 0;
+int returnAlertStub = 0;
 
 int networkAlertStub(float celcius) {

```

Comment:
This method is not used. Seems like unreachable code. If so then it needs to be cleaned up !